### ISSUE TITLE
extractAssets / getPageAssets doesn't work with vite's cssCodeSplit=false

### ISSUE THREAD

**Issue by vchirikov** [brillout/vite-plugin-ssr#644]
### Description

Vite has an option to extract all CSS into a single CSS file which called [cssCodeSplit](https://vitejs.dev/config/build-options.html#build-csscodesplit), if we set it to `false` the vite produces a css file, but vite-plugin-ssr doesn't inject this file into html (inside `build`, dev works fine, because it uses `<style>` tag)

### Error Message + Error Stack

`DEBUG=vps:extractAssets`:

<details> <summary> manifest with cssCodeSplit=false</summary>

```json
{
  "_chunk-d09dc09a.js": {
    "file": "assets/chunk-d09dc09a.js",
    "imports": [
      "_chunk-d6b0bc15.js"
    ]
  },
  "_chunk-d6b0bc15.js": {
    "file": "assets/chunk-d6b0bc15.js"
  },
  "_chunk-faf08fdb.js": {
    "file": "assets/chunk-faf08fdb.js"
  },
  "node_modules/.pnpm/vite-plugin-ssr@0.4.79_vite@4.1.1/node_modules/vite-plugin-ssr/dist/esm/client/entry.js": {
    "dynamicImports": [
      "pages/index.page.jsx",
      "pages/spa.page.client.jsx",
      "renderer/_default.page.client.jsx"
    ],
    "file": "assets/entry-server-routing.ced28eae.js",
    "imports": [
      "_chunk-faf08fdb.js"
    ],
    "isEntry": true,
    "src": "node_modules/.pnpm/vite-plugin-ssr@0.4.79_vite@4.1.1/node_modules/vite-plugin-ssr/dist/esm/client/entry.js"
  },
  "node_modules/.pnpm/vite-plugin-ssr@0.4.79_vite@4.1.1/node_modules/vite-plugin-ssr/dist/esm/client/router/entry.js": {
    "dynamicImports": [
      "pages/index.page.jsx",
      "pages/spa.page.client.jsx",
      "renderer/_default.page.client.jsx"
    ],
    "file": "assets/entry-client-routing.58fa9994.js",
    "imports": [
      "_chunk-faf08fdb.js"
    ],
    "isEntry": true,
    "src": "node_modules/.pnpm/vite-plugin-ssr@0.4.79_vite@4.1.1/node_modules/vite-plugin-ssr/dist/esm/client/router/entry.js"
  },
  "pages/index.page.jsx": {
    "file": "assets/pages/index.page.5fa7c29c.js",
    "imports": [
      "_chunk-d6b0bc15.js",
      "_chunk-d09dc09a.js"
    ],
    "isDynamicEntry": true,
    "isEntry": true,
    "src": "pages/index.page.jsx"
  },
  "pages/spa.page.client.jsx": {
    "file": "assets/pages/spa.page.client.b382759d.js",
    "imports": [
      "_chunk-d6b0bc15.js",
      "_chunk-d09dc09a.js"
    ],
    "isDynamicEntry": true,
    "isEntry": true,
    "src": "pages/spa.page.client.jsx"
  },
  "renderer/_default.page.client.jsx": {
    "file": "assets/renderer/_default.page.client.7206a042.js",
    "imports": [
      "_chunk-d6b0bc15.js"
    ],
    "isDynamicEntry": true,
    "isEntry": true,
    "src": "renderer/_default.page.client.jsx"
  },
  "style.css": {
    "file": "assets/style.1b4aa48d.css",
    "src": "style.css"
  }
}
```

</details>

[Repro repo](https://github.com/vchirikov/vite-plugin-ssr_tailwind-css)
```bash
pnpm i 
pnpm preview
```

**Comment by brillout** [comment]
Minimal reproduction: https://github.com/brillout/vps_cssCodeSplit-false

**Comment by brillout** [comment]
Fix released at `0.4.80`.

As usual if your company is up for it: https://github.com/sponsors/brillout.
